core/overlays/moc-prod/backend-prod/thoth-notification.yaml

```diff
@@ -19,7 +19,7 @@ spec:
             - "-H"
             - "Content-Type: application/json; charset=UTF-8"
             - "-d"
-            - "{'text':'I have successfully synchronized *Thoth Core* components to *CNV-PROD* *infra* namespace, see <https://argocd-server-argocd.apps.moc-infra.massopen.cloud/applications/prod-thoth-core-infra|ArgoCD UI> 🚀'}"
+            - "{'text':'I have successfully synchronized *Thoth Core* components to *PROD* *backend* namespace, see <https://argocd-server-argocd.apps.moc-infra.massopen.cloud/applications/prod-thoth-core-backend|ArgoCD UI> 🚀'}"
             - "$(THOTH_DEVOPS_WEBHOOK_URL)"
```

Comment:
Why was this file not only migrated environments but also moved from infra to backend namespace?

Reply:
That was an old issues, which was fixed in this.
As you can see the directory is named: **backend-prod**
The notification should relay it is syncing the **backend** namespace